includes/admin/upgrades/class-give-updates.php

```diff
@@ -676,7 +691,10 @@ public function __flush_resume_updates() {
 
 		// Reset counter.
 		$this->step = $this->percentage = 0;
-		++ $this->update;
+
+		$this->update = ( $this->get_total_db_update_count() > $this->update ) ?
```

Comment:
@DevinWalker Added a condition to prevent increase update counter.

Reply:
👍